setup.md for installation guide

FatimaArshad-DS opened the pull request:
Please review setup.md and inform about changes that need to be made. Please record this as my first contribution.

pgleeson commented:
Thanks for this @FatimaArshad-DS. Can you remove the changes to SimpleExample.ipynb from this PR? Also there is no need for a separate setup.md, as the installation info should be summarised in the README.

pgleeson commented:
Thanks for this @FatimaArshad-DS. Can you remove the changes to SimpleExample.ipynb from this PR? Also there is no need for a separate setup.md, as the installation info should be summarised in the README.

FatimaArshad-DS commented:
> Thanks for this @FatimaArshad-DS. Can you remove the changes to SimpleExample.ipynb from this PR? Also there is no need for a separate setup.md, as the installation info should be summarised in the README.

Thank you for responding. I have added another commit which doesn't have setup.md and added instructions in README. I didn't make any active changes in notebook. However, Ill check.

pgleeson commented:
Thanks @FatimaArshad-DS. The 3 changed files are still part of the PR. The simplest thing might be to make a new branch with just the changes to README.md and open a new PR with just that file.
